lib/spack/spack/build_systems/autotools.py

```diff
@@ -184,12 +189,6 @@ def patch(self):
             ``config.guess``
         """
```

Comment:
Remove it, otherwise it will always report that a patch was run. See #4103 for discussion on this.